### DISCUSSION TITLE
Is there a way to use remix with electron-vite?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [alex8088/electron-vite#481] [Q&A]
afaik, remix vite uses the `ssr` feature in vite which will split bundles across client/server clients. wondering if I could build an adapter for electron-vite that would plug into the config system that it requires, as I don't think there's currently a way to split the build of remix into individual bundles between server/client but I'm also pretty naive when it comes to vite tooling at the moment.

any suggestions would be helpful, thank you!

**Comment by rares-lupascu** [comment]
+1
